core/src/main/scala/filodb.core/metadata/Projection.scala

```diff
@@ -139,9 +139,13 @@ case class RichProjection(projection: Projection,
               rowKeyColIndices = newRkIndices,
               rowKeyType = newRkType)
   }
+
+  def getDataColumns(): Seq[Column] = {
```

Comment:
I would name this `dataColumns` and make it one line:

    def dataColumns: Seq[Column] = columns.collect { case d: DataColumn => d }